mythril/analysis/solver.py

```diff
@@ -97,9 +101,9 @@ def get_transaction_sequence(
     tx_constraints, minimize = _set_minimisation_constraints(
         transaction_sequence, constraints.copy(), [], 5000, global_state.world_state
     )
-
+    weighted = tx_constraints.weighted
     try:
-        model = get_model(tx_constraints, minimize=minimize)
+        model = get_model(tx_constraints, minimize=minimize, weighted=tuple(weighted))
```

Comment:
I would remove the intermediary variable. 

Also, does it ignore the weighted constraints that are still part of tx_constraints?

Reply:
Oh! Yes I should remove this param, some previous version of my code required it, now it makes things complicated